test/unit/spec-testing.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-next-hello-world
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+'use strict';
+
+import {Application, CoreBindings} from '@loopback/core';
+import Dredd = require('dredd');
+import {expect} from '@loopback/testlab';
+import {HelloWorldApp} from '../..';
+
+describe('Api Spec End Points', () => {
+  let dredd;
+  before(initEnvironment);
+
+  describe('input/output test', () => {
+
+    it('passes match', done => {
+      dredd.run((err, stats) => {
+        expect(err).to.be.eql(null);
+        expect(stats.failures).to.be.eql(0);
+        expect(stats.errors).to.be.eql(0);
+        expect(stats.skipped).to.be.eql(0);
```

Comment:
These three lines will fail the test with an error that make it difficult to immediately see which property was not zero. Here is a better solution:

```ts
expect(stats).to.containDeep({
  failures: 0,
  errors: 0,
  skipped: 0,
});
```